modules/@apostrophecms/rich-text-widget/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     icon: 'format-text-icon',
     label: 'apostrophe:richText',
     contextual: true,
+    placeholderText: 'apostrophe:richTextPlaceholder',
```

Comment:
@stuartromanek if we want to set a default value, looks like this:

![image](https://user-images.githubusercontent.com/8301962/191976329-d087cf71-2c6f-4c7c-ac09-1a51072c9099.png)

It is overridable on project side, so if a project sets `placeholderText: false` or any falsy value, it'll go back to the previous behavior:

![image](https://user-images.githubusercontent.com/8301962/191974777-363799a4-199c-4cc8-8734-bc2b5e7c56b5.png)

Reply:
I think we should keep a default in core.

'Start typing here...' is fine